---
int32 l_pulse_count
int32 r_pulse_count

